code/player/controller/DefaultWalkController.cs

```diff
@@ -2,24 +2,21 @@
 
 using Sandbox;
 
+using TTTReborn.Settings;
+
 namespace TTTReborn.Settings
 {
-    using Player;
-
     public partial class ServerSettings
     {
-        public bool IsSprintEnabled
-        {
-            get => DefaultWalkController.IsSprintEnabled;
-            set
-            {
-                if (DefaultWalkController.IsSprintEnabled == value)
-                {
-                    return;
-                }
+        public Topics.Movement Movement { get; set; } = new Topics.Movement();
+    }
 
-                DefaultWalkController.SetSprintEnabled(value);
-            }
+
+    namespace Topics
```

Comment:
I've seen this namespace a few times, what is the goal of describing what should be underneath it? `Topics` seems pretty vague to me.

Reply:
While `TTTReborn.Settings` stores the settings stuff, `TTTReborn.Settings.Topic` holds the setting topics, so you don't have just json structure stuff inside the defeault settings namespace.